### PR TITLE
Static pages( home, about, help and contact)

### PR DESCRIPTION
**Pull request by tunv**

### CODE REVIEW

File: app/assets/stylesheets/custom.css.scss [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+$gray-medium-light: #eaeaea;
+
+@mixin box_sizing {
+  -moz-box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+}
+
+
```
Comment (vuducdat) [inline app/assets/stylesheets/custom.css.scss]: convention !

Comment (phuongbk) [inline app/assets/stylesheets/custom.css.scss]: thừa 1 dòng nè.

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -1,2 +1,6 @@
 module ApplicationHelper
+  def full_title page_title = ""
+    base_title = "Ruby on Rails Tutorial Pilot App"
```
Comment (phuongbk) [inline app/helpers/application_helper.rb]: I18n

File: app/views/layouts/_shim.html.erb [hunk 3]
```diff
@@ -0,0 +1,4 @@
+<!--[if lt IE 9]>
+  <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/r29/html5.min.js">
+  </script>
+<![endif]-->
```
Comment (vuducdat) [inline app/views/layouts/_shim.html.erb]: you can use this file ?

File: app/views/layouts/_footer.html.erb [hunk 4]
```diff
@@ -0,0 +1,13 @@
+<footer class="footer">
+  <small>
+    The <a href="http://www.railstutorial.org/">Ruby on Rails Tutorial</a>
+    by <a href="http://www.michaelhartl.com/">Michael Hartl</a>
```
Comment (phuongbk) [inline app/views/layouts/_footer.html.erb]: I18n

File: config/routes.rb [hunk 5]
```diff
@@ -1,4 +1,10 @@
 Rails.application.routes.draw do
+  root 'static_pages#home'
+
+  get 'help' => 'static_pages#help'
+  get 'about' => 'static_pages#about'
+  get 'contact' => 'static_pages#contact'
```
Comment (vuducdat) [inline config/routes.rb]: ' ' -> " "

File: app/views/static_pages/about.html.erb [hunk 6]
```diff
@@ -0,0 +1,8 @@
+<% provide :title, t(".about") %>
+<h1><%= t ".about" %></h1>
+<p> 
+  The <a href="https://www.railstutorial.org/">Ruby on Rails Tutorial</a> is a 
+  project to make a book and screencasts to teach web development with 
+  <a href="http://rubyonrails.org/">Ruby on Rails</a>. This is the sample 
+  application for the tutorial.
```
Comment (phuongbk) [inline app/views/static_pages/about.html.erb]: I18n

File: app/views/layouts/_shim.html.erb (hunk 3 above)
Comment (tunv) [inline app/views/layouts/_shim.html.erb]: i think so!

File: config/application.rb [hunk 7]
```diff
@@ -31,5 +31,7 @@ class Application < Rails::Application
 
     # Do not swallow errors in after_commit/after_rollback callbacks.
     config.active_record.raise_in_transactional_callbacks = true
+
+    config.i18n.load_path += Dir[Rails.root.join('config', 'locales', '**', '*.{rb,yml}').to_s]
```
Comment (dieunb) [inline config/application.rb]: `"` not `'`

**Pull request merged**